workflow/step-conditions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Step Conditions (Coming Soon)"
+sidebarTitle: "Step Conditions"
+---
+
+**Step Conditions** allow you to customize a **Subscriber's** workflow journey using their **Data**, **Topics**, **Segment Membership**, or prior behavior.
```

Comment:
What a segment membership is? we don't have any concept like this

Reply:
Addressed.